getting-started/writing-in-yarn/tags-metadata.md

```diff
@@ -0,0 +1,104 @@
+# Tags and Metadata
+
+Yarn Spinner allows **tags** in nodes and lines. These tags (along with a few other things) make up the **metadata** of a node or line, which are never shown directly to the player.
+
+The primary purpose of metadata is to provide information to code that handles Yarn Spinner data and objects. Browse the [C# API](api/csharp) for the ways you can access this information from your code.
+
+## Tags in lines
+
+Tags are declared at the end of a line, and must always start with a *hash symbol* (#). With a few exceptions outlined in this page, they are mostly used by your own code, so it's up to you what content they should have, and how to handle them. Some example use cases are listed at the end of this page.
```

Comment:
Might be worth saying `Tags (also called hashtags) are declared` just because the term hashtag is used a fair bit in discussion